### ISSUE TITLE
hi,Have you compared the results of the code implementation and the paper?

### ISSUE THREAD

**Issue by HITjialanyu** [harryhan618/LaneNet#2]
And,In your experiment, which effect is better for lanenet and SCNN?

**Comment by harryhan618** [comment]
No. I don't have enough GPUs for training. Also, if I use mean shift for lane clustering from scikit-learn, it would be very slow. So, any ideas?
As to SCNN, you can check [this repository](https://github.com/harryhan618/SCNN_Pytorch). I think it works pretty good on corresponding dataset.

**Comment by AlexKaravaev** [comment]
@harryhan618 Have you tried [hdbscan](https://github.com/scikit-learn-contrib/hdbscan) for clustering? I was using it for whole another different project, but it had resulted in great performance, both in speed and accuracy.

**Comment by harryhan618** [comment]
@AlexKaravaev Actually I think mean shift works well. The bottleneck in my code is `estimate_bandwidth`. If the `bandwidth` of meanshift is predefined rather than `estimate_bandwidth`, I think it works fine. Maybe you can try a comparison experiment and exchange the idea with me. Thx!
